### PR TITLE
Drop support of OS file handles in gamelogic

### PR DESCRIPTION
**Pull request by slipher**
To make WebAssembly migration easier by getting rid of IPC calls which aren't purely sending bytes.

The feature is currently not used anywhere. There is one bit of code in the cgame which depends on PakPath::ReadFile, but that turns out not to be used also...

### CODE REVIEW

**Comment by slipher** [comment]
Fixes #164 by the way.

File: src/common/IPC/CommonSyscalls.h [hunk 1]
```diff
@@ -246,14 +246,14 @@ namespace VM {
 
     enum EngineFileSystemMessages {
         FS_INITIALIZE,
-        FS_HOMEPATH_OPENMODE,
+        FS_HOMEPATH_OPENMODE, // TODO(0.52) remove
```
Comment (illwieckz) [inline src/common/IPC/CommonSyscalls.h]: I prefer the `TODO:` syntax

**Comment by illwieckz** [comment]
If no one has something against the drop of that unused feature, this looks OK to me.

File: src/common/IPC/CommonSyscalls.h (hunk 1 above)
Comment (slipher) [reply src/common/IPC/CommonSyscalls.h]: I'm using `TODO(0.52)` as a search string to find stuff that I want to delete later. Do you have an idea what I should use instead?

Comment (illwieckz) [reply src/common/IPC/CommonSyscalls.h]: I usually look for the complete `TODO:` string so I’m not very happy with the missing colon, but that's not very important: those TODO are not expected to last. Make sure the related commit deleting this ends in 0.52 branch and it's OK. :grin:

**Pull request merged**